doc/releases/release-notes-3.2.rst

```diff
@@ -234,6 +234,25 @@ Architectures
 
 * RISC-V
 
+  * introduced support for RV32E
```

Comment:
Can you capitalize the start of the sentences? And dots at the ends if you feel like :-)